github-page/_docs/decorators/prop.md

```diff
@@ -541,20 +555,20 @@ class Something {
 
 // This would be mapped to
 {
-  type: [[[{ type: String }]]]
+  type: [[[{ type: String }]]];
```

Comment:
```suggestion
  type: [[[{ type: String }]]]
```

Reply:
Yup. It's VSCode and it's driving me crazy right now.

Scott

Reply:
nvm this is an wrong suggestion here